File: C++ Programming/Competitive Programming/Coding Challenges 2/program3.cpp
/*
Q. Accept number of rows and number of columns from user and display below
pattern.
Input : iRow = 3 iCol = 5
Output :A A A A A
        B B B B B
        C C C C C
*/

#include<iostream>
using namespace std;

void Pattern(int iRow, int iCol)
{
    int i=0;
    int j=0;

    for ( i = 1; i <= iRow; i++)
    {
            for ( j = 1; j <=iCol; j++)
            {
                cout<<char('A'+(i-1))<<"\t";
            }
        cout<<"\n";
    }
}

int main()
{
    int iValue1 = 0, iValue2 = 0;

    cout<<"Enter Number of Rows : ";
    cin>>iValue1;

    cout<<"Enter Number of Columns : ";
    cin>>iValue2;

    Pattern(iValue1, iValue2);

    return 0;
}

File: Java Programming/Competitive Programming/Coding Challenges 7/program3.java
/*
3. Accept number of rows and number of columns from user and display below
pattern.
Input : iRow = 5 iCol = 5
Output :$ * * * *
        # $ * * *
        # # $ * *
        # # # $ *
        # # # # $
*/

import java.util.*;

class program3
{
    public static void Pattern(int iRow,int iCol) 
    {
        int i=1,j=0;

        do {
           j=1;
            do {
                if (j<i) 
                    {
                        System.out.print("#\t");
                    } 
                else if(j==i)
                    {
                        System.out.print("$\t");
                    }
                else 
                    {
                        System.out.print("*\t");
                    }
                 j++;
               } while (j <= iCol);
               i++;
               System.out.println();   
        } while (i<=iRow);
    }

    public static void main(String[] args) 
    {
        Scanner sobj=new Scanner(System.in);

        int iValue1=0,iValue2=0;

        System.out.println("Enter number of rows :");
        iValue1=sobj.nextInt();

        System.out.println("Enter number of columns :");
        iValue2=sobj.nextInt();

        Pattern(iValue1, iValue2);

        sobj.close();
    }
}